Error building fits package

```
cp files/* "/rpmbuild"/SOURCES/
rpmbuild --define "_topdir "/rpmbuild"" --define "_etcdir "/src"/etc" --define "name fits" --define "version 0.10.1" -ba --clean "/rpmbuild"/package.spec
error: Bad source: /rpmbuild/SOURCES/fits-log4j.patch: No such file or directory
make: Leaving directory `/src'
make: *** [rpm-build] Error 1
make: *** [build] Error 2
```

--- a/rpm/fits/files/fits-log4j.patch
+++ b/rpm/fits/files/fits-log4j.patch
@@ -1,0 +1,11 @@
+--- log4j.properties	2016-03-25 23:36:12.386245658 +0100
++++ log4j.properties.orig	2016-03-25 23:35:37.195278070 +0100
+@@ -48,7 +48,7 @@
+ #
+ #------------------------------------------------------------------------------
+ log4j.appender.FILE = org.apache.log4j.RollingFileAppender
+-log4j.appender.FILE.File = ./fits.log
++log4j.appender.FILE.File = /var/log/archivematica/fits.log
+ log4j.appender.FILE.Threshold=ERROR
+ log4j.appender.FILE.Append = true
+ log4j.appender.FILE.MaxFileSize=10MB

